tests/conftest.py

```diff
@@ -21,6 +22,11 @@ def begin(conn):
         conn.execute('BEGIN')
 
 
+@pytest.fixture
+def event_loop():
+    return asyncio.get_event_loop()
+
```

Comment:
Why not creating a new fresh event loop? And then you can also add a finalizer to fixture to ensure event loop gets closed. E.g.

```python
@pytest.fixture()
def event_loop(request):
    """Create an asyncio event loop."""
    loop = asyncio.new_event_loop()
    asyncio.set_event_loop(loop)

    def teardown():
        loop.close()

    request.addfinalizer(teardown)

    return loop
```